daprieu/rare-client-cool-cake
# Pp create reaction

# Description

Admins can create new reactions. Normal users cannot access the page through the nav bar or the url.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Backend Branch:
```
git checkout pp-postReactions
```

Example image url:
```
https://res.cloudinary.com/nicecloudbro/image/upload/v1619800610/meIRL_ginncp.png
```

As an admin, go to the Reactions MGMT page, fill out the form, and submit.
As a user, you should not be able to access the page.

# Checklist:
